### ISSUE TITLE
Switch naming over for State* to Asset-hub

### ISSUE THREAD

**Issue by TarikGul** [paritytech/asset-transfer-api#212]
With the oncoming naming switch with common good parachains of the `specName` we need to ensure we switch over any instance of the following to the following:

```
statemint -> asset-hub-polkadot
statemine -> asset-hub-kusama
westmint -> asset-hub-westend
```

We need to ensure the switch is fluid with the runtime this is activated on. For anywhere where the specName is used we can ensure that we have a smooth transition by allowing all six `specName`'s as valid values.

**Comment by TarikGul** [comment]
rel: https://github.com/paritytech/asset-transfer-api/pull/237

**Comment by marshacb** [comment]
closed by: #237
